test/helpers/browser.js

```diff
@@ -2,7 +2,9 @@
 // Node module: loopback-boot
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
+/*eslint-env es6*/
```

Comment:
+1 for upgrading eslint-config-loopback.

For now, you can enable ES6 in `.eslintrc`, see https://github.com/strongloop/loopback-connector-kv-extreme-scale/blob/f2c9ac2813fc446b95b6f4e548826d74f5f93f95/.eslintrc#L3-L5 for an example